### ISSUE TITLE
Can it add item to the .ctv6?

### ISSUE THREAD

**Issue by AlexShyXie** [SchmidtPaul/CitaviR#12]
I want to add new record to the .ctv6, but I didn't find the function?

**Comment by SchmidtPaul** [comment]
No, sorry, I have not implemented that function yet and I don't think it would be easy to solve.
